packages/mdc-textfield/_mixins.scss

```diff
@@ -135,11 +135,13 @@
   @include mdc-text-field-helper-text-color_($mdc-text-field-disabled-helper-text-color);
   @include mdc-text-field-icon-color_($mdc-text-field-disabled-icon);
   @include mdc-text-field-fullwidth-bottom-line-color_($mdc-text-field-fullwidth-bottom-line-color);
+  @include mdc-text-field-fill-color_($mdc-text-field-disabled-background);
 
   pointer-events: none;
 
   .mdc-text-field__input {
-    border-bottom: 1px dotted;
+    // border-bottom-width: 1px;
```

Comment:
Remove this selector entirely if the styles are going away

Reply:
Completely right, I forgot to delete the commented code. Got it!